integrated_channels/moodle/exporters/content_metadata.py

```diff
@@ -103,7 +104,7 @@ def transform_start(self, content_metadata_item):
         """
         start_date = content_metadata_item.get('start', None)
         if start_date:
-            return int(parse(start_date).timestamp())
+            return int(parse(start_date).replace(tzinfo=timezone.utc).timestamp())
```

Comment:
my local system's timezone was causing these timestamps to return with a 5 hour offset in testing. i changed this to ensure we're always using a UTC datetime